src/components/Nav/Nav.tsx

```diff
@@ -43,6 +51,10 @@ export class Nav extends React.Component<INavProps, INavState> implements INav {
       this._selectedKey = this.props.initialSelectedKey;
     }
 
+    this._hasExpandButton = this.props.groups.some((group: INavLinkGroup, groupIndex: number) => {
```

Comment:
This can benefit from a comment. Are you saying if a group has a name or any children it should have the expand button?